datafusion/physical-expr/src/window/mod.rs

```diff
@@ -16,15 +16,15 @@
 // under the License.
 
 mod aggregate;
-mod built_in;
-mod built_in_window_function_expr;
 mod sliding_aggregate;
+mod udf;
```

Comment:
It is somewhat strange that we now have "UD ..." for user defined window functions as everything is a user defined window function.

Maybe we could use the names `StandardWindowExpr` and `StandardWindowFunctionExpr` instead

I think `WindowExpr` can be either an aggregate (like `sum`) or one of these standard window exprs (first_value, etc)